src/hooks/useUserAgent.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+
+type UserAgentPlatform = 'web' | 'android' | 'ios'
+
+export default function useUserAgent(): UserAgentPlatform {
+  const [device, setDevice] = useState<UserAgentPlatform>('web')
+
+  useEffect(() => {
+    const userAgent = window.navigator.userAgent
+    if (/android/i.test(userAgent)) setDevice('android')
+    if (/iPad|iPhone|iPod/.test(userAgent)) setDevice('ios')
+  }, [])
```

Comment:
We should use plain callback fn, not the hook, because this hook does nothing. It simply reads the `window.navigator.userAgent` (which won't mutate) and does regex test

Reply:
You're right, fixed in [3af6b6a](https://github.com/BigWhaleLabs/ketl-landing/pull/31/commits/3af6b6a421416e35a114feed69133a9bfcc6f01b).